09_json/willshen8/pkg/puppy/json/jsonifyPuppy.go

```diff
@@ -0,0 +1,22 @@
+package puppy
+
+import (
+	"encoding/json"
+
+	puppy "github.com/anz-bank/go-course/09_json/willshen8/pkg/puppy"
+	store "github.com/anz-bank/go-course/09_json/willshen8/pkg/puppy/store"
+)
+
+func UnmarshalPuppy(jsonData []byte, decodedPuppies *[]puppy.Puppy) {
+	unMarshalError := json.Unmarshal(jsonData, &decodedPuppies)
+	if unMarshalError != nil {
+		panic(unMarshalError)
+	}
+}
+
+func SaveJSONPuppyToStore(p *puppy.Puppy, m *store.MapStore) {
+	_, creationError := m.CreatePuppy(p)
+	if creationError != nil {
```

Comment:
`if err :=m.CreatePuppy(p); err != nil {`

Reply:
Thanks, still getting used to this short notation.